pom.xml

```diff
@@ -98,6 +99,11 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
```

Comment:
Do we have a standard about adopting dependencies?  I remember doing quite a bit of work inside AWS to convince myself/others that a dependency was acceptable.  Not saying anything about this one, never heard of it before, but have we done any investigation?

Reply:
We don't have a standard today. Created https://github.com/aws/event-ruler/issues/57 for it. 

On this library, I looked up this library based on this comment on the existing IntIntMap class https://github.com/aws/event-ruler/blob/main/src/main/software/amazon/event/ruler/IntIntMap.java#L12. The library was in active development and had good documentation. I see it used in some performance sensitive libraries so I gave it a try for Ruler.